creator-node/src/logging.js

```diff
@@ -3,13 +3,32 @@ const shortid = require('shortid')
 
 const config = require('./config')
 
+// taken from: https://github.com/trentm/node-bunyan/issues/194#issuecomment-347801909
+// since there is no official support for string-based "level" values
+// response from author: https://github.com/trentm/node-bunyan/issues/194#issuecomment-70397668
+function RawStdOutWithLevelName() {
+  return {
+    write: (log) => {
+      // duplicate log object before sending to stdout
+      const clonedLog = Object.assign({}, log)
+
+      // add new level (string) to level key
+      clonedLog.logLevel = bunyan.nameFromLevel[clonedLog.level]
+
+      const logLine = JSON.stringify(clonedLog, bunyan.safeCycles()) + '\n'
+      process.stdout.write(logLine)
+    }
+  }
+}
+
 const logLevel = config.get('logLevel') || 'info'
 const logger = bunyan.createLogger({
   name: 'audius_creator_node',
   streams: [
     {
       level: logLevel,
-      stream: process.stdout
+      stream: new RawStdOutWithLevelName(),
```

Comment:
i see one of the linked code examples does this `new fn()` thing but i don't get it at all
how do you instantiate a function?
i also see the other code examples do not have `new`

Reply:
interesting, this actually does work
```
  const fn = function () {
    return {
      'a': 5,
      action: () => 'action'
    }
  }
  const resp = new fn()
  console.log(resp)
```

but it also works without it - we should prob understand this line a bit

Reply:
ok, dug into this a bit more to understand, we should remove

Reply:
+1 sid here. this fn instantiation isn't common in our code base and we should follow patterns we have here. so, can we just pass in the fn without instantiating the ctor for it?